sql/core/src/test/scala/org/apache/spark/sql/execution/joins/BroadcastJoinSuite.scala

```diff
@@ -48,7 +48,7 @@ class BroadcastJoinSuite extends QueryTest with SQLTestUtils {
       .setMaster("local-cluster[2,1,1024]")
       .setAppName("testing")
     val sc = new SparkContext(conf)
-    spark = SparkSession.builder.getOrCreate()
```

Comment:
How about you fixing it in your PR? I will revert it back. Actually, this is not related to this PR. I just accidentally found this bug. : )